### ISSUE TITLE
Signothèque portable vectoriel

### ISSUE THREAD

**Issue by VV666** [VV666/LSF#1]
Bonjour, 
je crée un issues sur mon compte car ça ne concerne pas directement le projet [Signothèque](https://github.com/signotheque).

Avec une signothèque conséquente, il serait possible de créer un dictionnaire FR-LSF, basé sur des dessins vectoriels de signes.
Il existe déjà une multitude de dico de FR-LSF, vidéo notamment. Elix étant le plus complet et le plus récent (en terme de technologie employé). [Elix](http://www.elix-lsf.fr/), reposant sur la vidéo, permet d'avoir sur un PC fixe avec une bonne connexion à Internet, un outils de très bonne qualité.
Or, si vous utilisé Elix sur téléphone mobile et dans une zone mal couverte par la 3G (ce qui n'est pas rare ou que vous n'avez pas d'abonnement illimité à la 3G), Elix devient inutilisable. Le format vidéo ne se prête pas à zoomer, à faire des ralenti ou à être visionné sur de petit écrans. Et même dans de bonne condition, certaines personnes sont mal à l'aise avec la vidéo car parfois les signants signent vite. Dans ce cas, comprendre le signe devient difficile, ralentir floute la vidéo. Avoir une bonne perception de l'espace n'est pas non plus évident. (Cela dit, je trouve qu'Elix reste un outils formidable et il y a un sacré boulot de fait !)

Le format vectoriel permet de zoomer sans perte de qualité, est très léger et se prête très bien à une indexation (grâce aux méta-données). De plus, c'est un format compatible avec le web d'aujourd'hui.
Il devient donc envisageable de proposer un "complément" à Elix, mais destiné à être embarqué comme application sur un terminal mobile.

J'ai pas envie de faire une appli Android ou iOS. Par contre, une appli 100% web en HTML5, CSS3, SVG et JS serait possible. 
Ayant un Téléphone [FirefoxOS](https://fr.wikipedia.org/wiki/Firefox_OS), j'ai vu qu'il était possible de faire une application aussi ambitieuse en full web et qui fonctionne bien. Malheureusement, le projet FFOS est mort. Peut être renaîtra -t-il sous la forme de [B2G](https://wiki.mozilla.org/B2G) ... 
Mais le format web reste facilement portable sur tout les terminaux mobiles.

J'ai des croquis que je vais scanner et mettre en ligne.
C'est cette idée qui m'a motivé à rejoindre votre projet.

**Comment by VV666** [comment]
[20161130134300761.pdf](https://github.com/VV666/LSF/files/621774/20161130134300761.pdf)
Pas sûr que ce soit bien lisible et compréhensible.
Sur ce croquis, je n'ai pas précisé la recherche par configuration (inspiré par ce [site](http://lsfdico-injsmetz.fr/recherche-par-signe-de-la-main.php))

**Comment by Sungja** [comment]
Il me semble que les 2 projets s'entrecroisent quand même fortement... Effectivement, pour en faire un dictionnaire, il faudrait avoir un stock conséquent par rapport à une signothèque (le concept étant nouveau, j'imagine que personne n'a de format en tête, contrairement au dictionnaire).

**Comment by VV666** [comment]
Complètement. J'ai besoin de la signothèque pour faire un dico, faire sans, c'est ré-inventer la roue ! ^^

**Comment by Sungja** [comment]
Juste que le mot "signothèque" n'évoque rien d'existant et d' éprouvé. Alors qu'on a tous une image, une référence dans la tête quand on dit/lit le mot "dictionnaire".
Est ce que tu connais : http://lsf.wikisign.org/wiki/Wikisign:Accueil  ?

**Comment by VV666** [comment]
Tout à fait, c'est pas très concret pour tout le monde. Alors que ça ce comprend très bien : Signe + thèque qui vient du grec et qui peut être traduit par ranger : là où on range les signes, une collection de signes en somme.
Perso, ça m'a parlé tout de suite :-)

Je connais wikisign. Mais là, je suis au boulot et c'est bloqué ... Oui oui, j'ai pas accès à Wikisign ... Par contre j'ai accès à jeuxvideo.com ... Je ne sais pas si je dois rire ou pleurer ... -_-'
